src/inet/tests/Makefile.am

```diff
@@ -205,11 +205,11 @@ TestInetLayerMulticast_LDADD                          = libTestInetCommon.a $(CO
 #
 
 NLFOREIGN_FILE_DEPENDENCIES                           = \
-   $(top_builddir)/src/include/CHIPVersion.h            \
    $(CHIP_LDADD)                                        \
    $(NULL)
 
 NLFOREIGN_SUBDIR_DEPENDENCIES                         = \
+   $(top_builddir)/src/include                          \
```

Comment:
Any motivation behind behind less rather than more specific about the dependency here?

Reply:
the specific dependency was not working because the C preprocessor wasn't finding that file during generation of the .po files